components/popover/index.js

```diff
@@ -123,10 +123,10 @@ export class Popover extends Component {
 
 	setOffset() {
 		const { anchor, popover } = this.nodes;
-		const { parentNode } = anchor;
-		if ( ! parentNode ) {
+		if ( ! anchor || ! anchor.parentNode ) {
 			return;
 		}
+		const { parentNode } = anchor;
```

Comment:
Both `anchor.parentNode` and `{ parentNode } = anchor` in one place feels a bit weird to me.

Reply:
yep we could avoid destructuring